module-4/backend/week_four.md

```diff
@@ -4,8 +4,32 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-TBD  
+* What's the purpose of a project management tool?
+<p>To help coordinate and organize tasks and responsibilities for members of the team.</p>
+* What are the stages of a card when using Pivotal Tracker?
+<p>Cards start in the icebox, once someone is assigned to them and they begin working on it, it moves to the started phase. Once the card is complete it moves to the finish phase , once a PR has been merged it is delivered. The Project Owner then reviews what is in production and either accepts or rejects the card, depending on whether or not the expected functionality in production meets what the card requires. </p>
+* What are the four types of cards available in Pivotal Tracker?
+<p>Feature, bug, chore, release</p>
+* What's your favorite project management tool?
+<p>I like waffle, mostly because I know how to use it really well. I can see the benifits that Pivotal Tracker has over Waffle in terms of added functionality</p>
+* Why do we create staging environments?
+<p>To ensure that our code will run correctly in a production environment</p>
+* What are the characteristics of a good README (in your opinion)?
+<p>A good README documents how to install and use an application. The specific type of information in a README depends on the type of application, for example an API will often have information about each endpoint and how to access and get/post/patch data.</p>
+* What's one main improvement you're going to make to your code regarding accessibility issues?
+<p>I'm much more aware of how important tabbing around a page can be and intend to implement it going forward.</p>
+* What are some basic security concerns to be aware of when building applications?
+<p>SQL injection, protecting things like API keys.</p>
+* Why is continuous integration helpful/important?
+<p>Continuous Integration is important in terms of confirming that your application is working the way you expect it to. CI applications run tests for you everytime you push a branch. This is helpful in situations in which your test suite takes a long time to run, you can push a branch have your CI run the tests and continue working. </p>
```

Comment:
it also helps ensure that the application can run with different versions of ruby or gems in general 👍